### ISSUE TITLE
method to convert CollectionMetadata to STAC json

### ISSUE THREAD

**Issue by jdries** [Open-EO/openeo-python-client#344]

**Comment by soxofaan** [comment]
I'm not sure what you mean here, as `CollectionMetadata` in the client has no `as_dict` method.
And the class is basically just a wrapper around STAC(ish) metadata given to the constructor, with accessor methods and properties for common metadata items;  it's not really a builder for metadata

**Comment by jdries** [comment]
my bad, commits got lost, I now fixed it without requiring something in the client
But anyway: for tests but also for user logs, it would be convenient that we can export our metadata into a standardized format. 
The most logical one being:
https://github.com/stac-extensions/datacube

**Comment by soxofaan** [comment]
indeed, we could indeed crank up STAC support/compatibility in a couple of places. Also related: 
- #133
